kloppy/domain/services/transformers/dataset.py

```diff
@@ -292,18 +299,21 @@ def __flip_frame(self, frame: Frame):
                 other_data=data.other_data,
             )
 
-        return Frame(
-            # doesn't change
-            timestamp=frame.timestamp,
-            frame_id=frame.frame_id,
-            ball_owning_team=frame.ball_owning_team,
-            ball_state=frame.ball_state,
-            period=frame.period,
-            # changes
-            ball_coordinates=self.flip_point(frame.ball_coordinates),
-            players_data=players_data,
-            other_data=frame.other_data,
+        frame = create_frame(
```

Comment:
The factory shouldn't be used here. The `statistics` needs to be passed here to `Frame`